packages/trpc/src/testutils.ts

```diff
@@ -4,6 +4,24 @@ import type { AppRouter } from "./index";
 import superjson from "superjson";
 import { faker } from "@faker-js/faker";
 
+export function getRecipeStats(userId: string) {
+  return {
+    userId,
+    title: faker.string.alphanumeric(10),
+    description: faker.string.alphanumeric(10),
+    yield: faker.string.alphanumeric(10),
+    folder: "main",
+    activeTime: faker.string.alphanumeric(10),
+    totalTime: faker.string.alphanumeric(10),
+    source: faker.string.alphanumeric(10),
+    url: faker.string.alphanumeric(10),
+    notes: faker.string.alphanumeric(10),
+    ingredients: faker.string.alphanumeric(10),
+    instructions: faker.string.alphanumeric(10),
+    rating: faker.number.int({ min: 1, max: 5 }),
+  };
+}
```

Comment:
Generators like this should be called `factories`. We should create a folder for testutils and a subfolder called `factories`. This would reside in that directory as recipeFactory.ts